.readthedocs.yml

```
@@ -10,21 +10,24 @@ sphinx:
   configuration: docs/source/conf.py
   # fail_on_warning might generate hard to fix error, in this case it can be
   # disabled but this also means those errors will fail silently, choose wisely.
-  fail_on_warning: true
+  fail_on_warning: false
 
 # Build documentation with MkDocs
 #mkdocs:
 #  configuration: mkdocs.yml
 
 # Optionally build your docs in additional formats such as PDF and ePub
-formats: all
+formats: []
 
-# Optionally set the version of Python and requirements required to build your docs
-#python:
-#  version: 3.6
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "mambaforge-22.9"
 
 conda:
-  environment: environment-docs.yml
+  environment: environment.yml
```

This should stay the same, no?
```suggestion
  environment: environment-docs.yml
```